merqube_client_lib/templates/equity_baskets/create_index.py

```diff
@@ -0,0 +1,54 @@
+"""
+Create an equity basket index
+"""
+import logging
+import os
+from typing import Final
+
+import click
+
+from merqube_client_lib.logging import get_module_logger
+from merqube_client_lib.templates.equity_baskets.decrement.create import (
+    create as dec_index,
+)
+from merqube_client_lib.templates.equity_baskets.multiple_no_corax.create import (
+    create as mult_index,
+)
+from merqube_client_lib.templates.equity_baskets.single_stock_total_return_corax.create import (
+    create as ss_index,
+)
+
+SUPPORTED_INDEX_TYPES: Final[list[str]] = ["decrement", "single_stock_total_return", "multi_basket_no_corax"]
+
+logger = get_module_logger(__name__, level=logging.DEBUG)
+
+
+@click.command()
```

Comment:
since we were basically copy-pasta-ing this into each template, i deleted all that and moved it here with the switch on `func`.